ngraph/frontend/paddlepaddle/src/frontend.cpp

```diff
@@ -49,8 +49,15 @@ NamedOutputs make_ng_node(const std::map<pdpd::TensorName, Output<Node>>& nodes,
             named_inputs[input_port.parameter()].push_back(node_it->second);
         }
     }
+    NamedOutputs outputs;
+    // In case the conversion function throws exception
+    try {
+        outputs = creator_it->second(NodeContext(DecoderPDPDProto(op_place), named_inputs));
```

Comment:
Is it possible to create some tests for this?

Reply:
Yep, it needs to be tested. I think we can add an example that triggers an exception in op conversion